docs/source/overview.rst

```diff
@@ -195,7 +195,7 @@ dimensionality of the referenced data.
 		tag = block.create_tag('interesting part', 'nix.roi', position)
 		tag.extent = extent
 		# finally, add the referenced data to this tag
-		tag.references.add(data)
+		tag.references.extend(data)
```

Comment:
append is for single key and extend is for multiple keys.
I think data should be given to extend() but both will work.